lib/spans/span-streamer.js

```diff
@@ -8,10 +8,10 @@
 const logger = require('../logger').child({component: 'span-streamer'})
 const NAMES = require('../metrics/names').INFINITE_TRACING
 
+const SPAN_DROP_MSG_INTERVAL_MS = 30000
```

Comment:
In practice, 5 seconds is more than we need. For a situation where the queue struggles to drain, this gets real noisy and the increased interval doesn't add much value given the other metrics we can check.